pkg/services/ngalert/schedule_test.go

```diff
@@ -26,7 +27,8 @@ func TestAlertingTicker(t *testing.T) {
 	t.Cleanup(registry.ClearOverrides)
 
 	mockedClock := clock.NewMock()
-	ng.schedule = newScheduler(mockedClock, time.Second, log.New("ngalert.schedule.test"), nil)
+	schefCfg := schedulerCfg{c: mockedClock, baseInterval: time.Second, logger: log.New("ngalert.schedule.test"), evalApplied: nil, evaluator: eval.Evaluator{Cfg: ng.Cfg}}
```

Comment:
Nit, but writing it over several lines makes it more readable.
```suggestion
	schefCfg := schedulerCfg{
		c: mockedClock, baseInterval: time.Second, logger: log.New("ngalert.schedule.test"), 
		evalApplied: nil, evaluator: eval.Evaluator{Cfg: ng.Cfg},
	}
```